### PR TITLE
Added sleeping time in minutes for easier change, QoL addition on time for next battle, added discord link

### PR DESCRIPTION
**Pull request by kromo-s**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -189,8 +190,8 @@ const sleepingTime = 1800000;
         } catch (e) {
             console.log('Routine error at: ', new Date().toLocaleString(), e)
         }
-        await console.log(process.env.ACCOUNT,'waiting for the next battle in ', sleepingTime / 1000 / 60, ' minutes')
-        await console.log('If you need support for the bot join the telegram group https://t.me/splinterlandsbot and dont pay scammers')
+        await console.log(process.env.ACCOUNT,'waiting for the next battle in',sleepingTime / 1000 / 60,'minutes at', new Date(Date.now() +sleepingTime).toLocaleString() )
```
Comment (alfficcadenti) [inline index.js:193]: can you please add the space before and after the minutes in the console.log?

**Review by alfficcadenti — CHANGES_REQUESTED** [review]
thanks for the PR, left just 1 comment

File: index.js (hunk 1 above)
Comment (Hinetion) [reply index.js:193]: `        await console.log(process.env.ACCOUNT,'waiting for the next battle in', sleepingTime / 1000 / 60 , 'minutes at' , new Date(Date.now() +sleepingTime).toLocaleString() )
`
Like this?

Comment (alfficcadenti) [reply index.js:193]: no i meant in the console message so:
sleepingTime / 1000 / 60, ' minutes at ', new Date(Date.now() +sleepingTime).toLocaleString() )

**Review by alfficcadenti — APPROVED** [review]

**Pull request merged**